source/views/main.js

```diff
@@ -769,6 +769,12 @@ gestureStart: function(inSender, e){
 	this.previousScale = e.scale;
 	this.previousS = 0;
 
+	e.centerX = 0;
```

Comment:
Changed this to only calculate a centerX/centerY when they are not already present.